vnext/DeforkingPatches/ReactCommon/turbomodule/samples/SampleTurboCxxModule.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "SampleTurboCxxModule.h"
```

Comment:
This file is forked only because the Facebook version uses #import instead of #include.  I expect that is something Facebook will change when they want TurboModules to be widely consumed.

Reply:
Wait, was it an `#import` or an `import`? The former is garbage. The latter is header unit syntax for C++ 20 modules that we can support once we're on a newer compiler. See https://devblogs.microsoft.com/cppblog/c-modules-conformance-improvements-with-msvc-in-visual-studio-2019-16-5/